packages/material/src/complex/MaterialTableControl.tsx

```diff
@@ -35,7 +35,27 @@ const EmptyTable = () => (
 const TableHeaderCell = ({cellProperty}) =>
   <TableCell >{cellProperty}</TableCell>;
 
-const TableContentCell = ({rowPath, cellProperty, cellPath, errors, resolvedSchema}) => {
+const TableWithContent = tableProps => {
```

Comment:
Is this a duplicate of line 86?

Reply:
I think I already updated the PR while you were reviewing it, I don't see the change you are referring to here.